apps/core/scenes/search/Search.js

```diff
@@ -234,51 +93,11 @@ const styles = StyleSheet.create({
   },
   bottomSpacer: {
     flex: 1,
-    web: {
```

Comment:
Sorry about this 😄. But can we at the end return this style and remove that new `{Platform.OS !== 'web'` condition for `bottomSpacer`.

After I tested some smaller resolutions I think that it looks better when there is at least little bit of padding at the bottom.
I this version (in extreme cases) when I scroll down it looks like this and I cannot scroll further.
<img width="833" alt="Screenshot 2019-04-16 at 16 40 56" src="https://user-images.githubusercontent.com/2660330/56219402-f8e1f780-6066-11e9-958e-a0ca897b4b58.png">

It's better to let user scroll little bit more and add some breathing space to button at the bottom like in current master.

<img width="835" alt="Screenshot 2019-04-16 at 16 41 36" src="https://user-images.githubusercontent.com/2660330/56219497-2333b500-6067-11e9-98fa-247936dbe24e.png">